normandy/studies/models.py

```diff
@@ -8,6 +8,9 @@ class Extension(models.Model):
     name = models.CharField(max_length=255)
     xpi = models.FileField(upload_to='extensions')
 
+    class Meta:
+        ordering = ('name',)
```

Comment:
What do you think about making this `-id` instead of `name`? I think that would better match the other APIs, and would also nicely fix #1091.

Reply:
amended